README.md

```diff
@@ -12,6 +12,10 @@ This a personal  portfolio project to make us familiar with tools(like **Figma**
 - CSS
 - Figma template
 
+## Live Demo
+
+[Click here to view Live website Demo](https://mwenyoa.github.io/Portfolio/)
+
 ## Getting Started
 To get a local copy up and running follow these simple example steps.
```

Comment:
**You did the requirement! Your project is perfectly working! 👏👏👏**

- [ ] [OPTIONAL] However, I strongly recommend you to fill all links as working conditions in your Readme.MD file. Your LICENSE link seems not working somehow, if you could at the LICENSE to your project it would be a great thing for your Portfolio!

Reply:
@Rustamxon7 
Thanks a lot for the feedback, and the recommendation is well noted.